src/Lecture1.hs

```diff
@@ -63,7 +65,8 @@ sumOfSquares x y = error "TODO!"
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integer -> Integer
+lastDigit n = if ((abs n) < 10) then abs n else lastDigit (mod (abs n) 10)
```

Comment:
I believe, you don't need recursion here as `mod (abs n) 10` already gives you the last digit of any number 👍🏻